04_numeronym/davidbroughsmyth/main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+)
+
+var out io.Writer = os.Stdout
+
+func numeronyms(vals ...string) []string {
+	ook := make([]string, len(vals))
```

Comment:
nitpick , it is not very obvious to me what the output variable ook means, is it just a random name in which case would you consider something more obvious like out/result/x  etc

Reply:
Thanks for the review. I wondered if anyone would pick that up, it's my goto var name for simple functions like this. I should have changed it before committing.

Reply:
Please revise your commits according to your reviewers comments. Then re-add the "Ready for review" label.

Reply:
Fixed my strange variable name and added a few more tests.